### DISCUSSION TITLE
Trying to add store: FNAC receive 403 error (works in browser)

### DISCUSSION THREAD

**Discussion by johnneerdael** [jef/streetmerchant#1092] [Help]
Hi everyone, 

I'm trying to set up a new store for the French website (FNAC) to monitor the PS5, however I keep getting 403 errors, although the links do work when I use my browser? I have changed my user agent in my .env file to match my actual user agent string.

```import {Store} from './store';

export const Fnac: Store = {
	labels: {
		inStock: {
			container: '.f-buyShopBox',
			text: ['En stock en ligne']
		},
		maxPrice: {
			container: 'span[class*="PriceString"]',
			euroFormat: true
		},
		outOfStock: {
			container: '.f-buyShopBox',
			text: ['Indisponible en ligne']
		}
	},
	links: [
		{
			brand: 'sony',
			cartUrl: 'https://secure.fnac.com/basket/add?from=111&productId=14119956',
			model: 'ps5 console',
			series: 'sonyps5c',
			url: 'https://www.fnac.com/Console-Sony-PS5-Edition-Standard/a14119956/w-4'
		}
	],
	name: 'fnac'
};
```

Any tips or pointers? Opening the URL in my browser works just fine.

**Comment by anthonymq** [comment]
Add your USER_AGENT in the .env file.
Ex: Mozilla/5.0 (Macintosh; Intel Mac OS X 11_0_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.67 Safari/537.36

**Comment by johnneerdael** [comment]
It seems it sometimes is working:
[17:09:13] warn :: ✖ [fnac] [sony (sonyps5c)] ps5 console :: STATUS CODE ERROR 403
[17:09:59] info :: ✖ [fnac] [sony (sonyps5c)] ps5 console :: OUT OF STOCK

I expect its some sort of refresh protection, is it possible to set a custom sleep timer per 'site' instead of global in .env?
Meaning, make fnac refresh slow while others go faster.

**Comment by anthonymq** [comment]
Same thing here, I think I've seen someone talk about a refresh rate per site. Trying to find it...

**Comment by anthonymq** [comment]
In the STORES env variable :
Both supported formats are comma separated

1. Standard E.g.: "nvidia"

2. Advanced E.g: STORE:PAGE_SLEEP_MIN:PAGE_SLEEP_MAX, E.g: nvidia:10000:30000

Default: nvidia
